src/msg/async/net_handler.cc

```diff
@@ -127,6 +129,7 @@ void NetHandler::set_priority(int sd, int prio, int domain)
 {
   if (prio >= 0) {
     int r = -1;
+    IGNORE_UNUSED(r);
```

Comment:
instead of using `(void)r`, we can limit the scope of `r` and define it when it is firstly initialized.